targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCommon/Private/PlayFabCommmonUtils.cpp.ejs

```diff
@@ -1,9 +1,27 @@
-﻿<%- copyright %>
+<%- copyright %>
 
 #include "PlayFabCommonUtils.h"
 
 using namespace PlayFabCommon;
 
+// see {your unreal root folder}\Engine\Source\Runtime\Launch\Resources\Version.h for more preprocessor defines
+#if ENGINE_MINOR_VERSION==20
+FString PlayFabCommonUtils::GetTempDir()
+{
+    FString vars[] = { TEXT("TEMPDIR"), TEXT("TEMP"), TEXT("TMP") };
+    TCHAR FileSystemPath[256];
```

Comment:
This is used in 3 places. Please define a local const and use it to futureproff out of sync issues